air/tracing_benches.sh

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env bash
+set -o pipefail -e
+#
+# This command executes AquaVM on the same data as benches, but outputs a trace.
+# Its dependencies:
+#   + jq utility
+#   + compiled AquaVM WASM binary
+#   + installed air-trace
+
+function trace_len {
+    jq '.trace | length'
+}
+
+function jq_prepend_trace {
+    jq -c ".trace |= [ $1 ] + ."
+}
+
+function insert_left {
+    local input="$1"
+    local len=$(trace_len < "${input}")
+
+    jq_prepend_trace "{par: [$len, 0]}" < "${input}"
+}
+
+function insert_right {
+    local input="$1"
+    local len=$(trace_len < "${input}")
+
+    jq_prepend_trace "{par: [0, $len]}" < "${input}"
+}
+
+function error {
+    echo "$1" >&1
+    exit "${2:-1}"
+}
+
+function main {
+    if [ "$#" \< 2 ]; then
+        if [ x"$1" == x"--help" ]; then
+            error "usage: $0 (--all|big|long) (--all|merge|nomerge) [ air-trace run args ... ]"
+        else
+            error "Provide at least two arguments: (--all|big|long) and (--all|merge|nomerge)"
+        fi
+    fi
+    local tests
+    local subtests
+    case $1 in
+        big|long)
+            tests=( "$1" )
+            ;;
+        --all)
+            tests=( big long )
+            ;;
+        *)
+            error "Unknown test type: $1"
+            ;;
+    esac
+    case $2 in
+        merge|nomerge)
+            subtests=( "$2" )
+            ;;
+        --all)
+            subtests=( merge nomerge )
+            ;;
+        *)
+            error "Unknown subtest type: $2"
+            ;;
+    esac
+    shift 2
+
+    : "${AIR_WASM_RUNTIME_PATH:=../target/wasm32-wasi/release/air_interpreter_server.wasm}"
```

Comment:
Maybe `AIR_INTERPRETER_WASM`? Don't like the `runtime` word here. Also the binary name is `interpreter`, not `runtime`

Reply:
This is variable used by the air-trace, and is to be renamed with the related option.

Reply:
Yep. Let's change it now in both script and air-trace, or late in a separate PR